README.md

```diff
@@ -11,12 +11,16 @@ any kind of deployment environment.
 
 ## Concourse Lite
 
-This runs outside an environment and is responsible for creating and
-destroying a VPC containing Bosh and Concourse.
+Concourse Lite is responsible for bootstrapping a full Bosh and Concourse 
+environment ([paas-cf][]). First it creates a Vagrant instance running on AWS.
+Second this Vagrant instance is used to provision the full environment.
```

Comment:
This instance isn't used for anything other than provisioning the Bosh and Concourse. Once that step is completed, this is no longer used.